cmake/compiler/gcc/target.cmake

```diff
@@ -68,6 +68,10 @@ elseif("${ARCH}" STREQUAL "arc")
   list(APPEND TOOLCHAIN_C_FLAGS
     -mcpu=${GCC_M_CPU}
     )
+elseif("${ARCH}" STREQUAL "sparc")
+  list(APPEND TOOLCHAIN_C_FLAGS
+    -mcpu=leon -mflat
```

Comment:
Don't add -mflat twice.

Reply:
I've moved `-mflat` from `arch/sparc/CMakeLists.txt` and `cmake/compiler/gcc/target.cmake` to `cmake/compiler/gcc/target_sparc.cmake` like arm and x86.